tests/unit/general/resources/images-parametric-tests.json

```
@@ -2,16 +2,19 @@
   "parameters": [
     {
       "image_property": "ingressNginx.controllerChroot",
+      "helmfile_selector": "-lapp=ingress-nginx",
```

```suggestion
      "helmfile_selector": "app=ingress-nginx",
```
And add the flag during invocation.

Also if you want to be more specific you have the built in labels `name`, `namespace`, and `chart` to reduce the number of templates needed to be generated.

---

Done 3c79a32e

---

Recheck this, you have split the flag in two instead of the flag from the labels.